_posts/2023-01-26-rust-32.0.0.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: "January 2023 Rust Apache Arrow Highlights"
+date: "2023-01-26 00:00:00 +0000"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Introduction
+
+With the recent release of [32.0.0](https://crates.io/crates/arrow/32.0.0) of the Rust implementation of [Apache Arrow](https://arrow.apache.org/), it seemed timely to highlight some of the communities work since the [last update](https://arrow.apache.org/blog/2022/06/16/rust-16.0.0/).
+
+The most recent list of detailed changes can always be found in the [CHANGELOG](https://github.com/apache/arrow-rs/blob/master/CHANGELOG.md), with the full historical list available [here](https://github.com/apache/arrow-rs/blob/master/CHANGELOG-old.md).
+
+# Arrow
+
+[arrow](https://crates.io/crates/arrow) and [arrow-flight](https://crates.io/crates/arrow-flight) are native Rust implementations of Apache Arrow. Apache Arrow defines a language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware like CPUs and GPUs. The Arrow memory format also supports zero-copy reads for lightning-fast data access without serialization overhead.
+
+The [Rust language](https://www.rust-lang.org/) offers best in class performance,  memory safety, and the developer productivity of a modern programming language. These features make Rust an excellent choice for building modern high performance analytical systems. When combined, Rust and the Apache Arrow Ecosystem are a compelling toolkit for building the next generation of systems.
+
+The [repository](https://github.com/apache/arrow-rs) recently passed 1400 stars on github, and the community has been focused on performance and feature completeness.
+
+
+**Major Highlights**
+
+* **New CSV and JSON Readers**: The CSV and JSON readers have been revamped. Their performance has more than doubled, and they now support push-driven parsing, and better streaming object storage integration.
```

Comment:
```suggestion
* **New CSV and JSON Readers**: The CSV and JSON readers have been revamped. Their performance has more than doubled, and they now support push-driven parsing facilitating async streaming decode from object storage.
```

I think we should highlight why you would care about push-driving parsing :smile: